### PR TITLE
OFC and AudioTrigger notifications

### PR DESCRIPTION
**Pull request by rbovill**

### CODE REVIEW

File: src/org/lsst/ts/jenkins/components/Csc.groovy [hunk 1]
```diff
@@ -61,6 +62,7 @@ def email() {
                 "ts-mtrotator": "ttsai@lsst.org",
                 "ts-observatory-control": "tribeiro@lsst.org",
                 "ts-observatory-model": "tribeiro@lsst.org",
+                "ts-ofc": "tribeiro@lsst.org, ttsai@lsst.org",
```
Comment (couger01) [inline src/org/lsst/ts/jenkins/components/Csc.groovy:65]: You need to remove the space between the comma otherwise it won't work.

**Review by couger01 — CHANGES_REQUESTED** [review]
Only one fix otherwise LGTM.

File: src/org/lsst/ts/jenkins/components/Csc.groovy (hunk 1 above)
Comment (rbovill) [reply src/org/lsst/ts/jenkins/components/Csc.groovy:65]: Syntax change is not necessary in this case.

**Review by couger01 — APPROVED** [review]
LGTM.

**Pull request merged**